eisbuk-admin/src/AppContent.tsx

```diff
@@ -21,6 +21,7 @@ import CustomerAreaPage from "@/pages/customer_area";
 import useFirestoreSubscribe from "@/hooks/useFirestoreSubscribe";
 
 import { queryOrganizationStatus } from "@/store/actions/authOperations";
+import AttendanceSheet from "./components/atoms/AttendanceSheet/AttendanceSheet";
```

Comment:
This is the cause of `/attendance_sheet` failing, you need to import a page component, not the component rendered on the page.